### ISSUE TITLE
Way to discuss about new ideas

### ISSUE THREAD

**Issue by petriuslima** [php-wpi/about#2]
I really like the idea of initiative, but I would like too if the the ideas proposed were open. Can you provide a way to share and discuss this ideas?

It can be a list in README, a file management like PROPOSALS.md + folder PROPOSALS with each one specified and ACTIVED_PROJECTS.md + folder or just an issue thread (or specific repository).

What do you think?

**Comment by gabrielrcouto** [comment]
Good point @petriuslima, I'll add to README an instruction about how to share and discuss ideas.

I think is better for everyone that each proposal be submited as an Issue (with the label "idea"). After a long discussion, we can use another label (like "vote") for public voting, and "in production" to show what we are developing. And, of course, on the README we will keep a list, divided by the idea status, of each issue.

Is it good for you?

**Comment by petriuslima** [comment]
Perfect! 👌
